omise/controllers/front/payment.php

```diff
@@ -0,0 +1,28 @@
+<?php
```

Comment:
@nimid if it's not break any PrestaShop rules, is it possible to have file name to `OmisePaymentModuleFrontController.php` to reflect its class name?